roles/gateway_preupgrade_analyzer/README.md

```diff
@@ -37,7 +40,6 @@ Dependencies
 ------------
 There are no dependencies upon other roles.
 
-
 Example Playbook
 ------------------
 file: playbooks/gateway-preupgrade-analyzer.yml
```

Comment:
please update the path in the example playbook for the report

Reply:
fixed